app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,63 @@
+class PostsController < ApplicationController
+  before_action :set_post, only: [:show, :edit, :update, :destroy]
+  before_filter :authenticate_user!
+  # GET /posts
+  # GET /posts.json
+  def index
+    @posts = Post.all.includes(:user)
+  end
+
+  # GET /posts/1
+  # GET /posts/1.json
+  def show
+  end
+
+  # GET /posts/new
+  def new
+    @post = Post.new
+  end
+
+  # GET /posts/1/edit
+  def edit
+  end
+
+  # POST /posts
+  # POST /posts.json
+  def create
+    @post = Post.new(post_params)
+    @post.user_id = current_user.id
+    if @post.save
+      redirect_to posts_url, notice: 'Post was successfully created.'
+    else
+      render :new
+    end
+  end
+
+  # PATCH/PUT /posts/1
+  # PATCH/PUT /posts/1.json
+  def update
```

Comment:
It looks like you took out the jsony stuff that was going on (especially since you removed jbuilder from the gemfile and didn't replace it with anything). If that's the case, you should also take out the json comments as well.